CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 ##### Fixes :wrench:
 
+- Fixed any C++ and shader errors that occur when building under Unreal Engine 5.4
```

Comment:
Usually we count support for new UE versions as an "addition" instead of a fix. You can see the changelog [here](https://github.com/CesiumGS/cesium-unreal/blob/main/CHANGES.md#v1310---2023-09-20) for what we said for 5.3.

Reply:
Makes sense! I was actually torn as to which section, when I put that in.

Reply:
Change committed